### PR TITLE
DCT-315

### PR DESCRIPTION
**Pull request by EBEREGIT**
### Error Message 109

- Created a new sample 
- Added code sample to the documentation

### CODE REVIEW

**Review by TheChronicMonster — CHANGES_REQUESTED** [review]

File: docs/src/rsh/errors/index.md [hunk 1]
```diff
@@ -2039,10 +2039,32 @@ can either abstract the arguments into a new `{!rsh} Data` type or make separate
 ## {#RE0109} RE0109
 
 This error indicates that there are multiple `{!rsh} Participant`s or `{!rsh} ParticipantClass`es
```
[the remark this answers is not in the document]
Comment (EBEREGIT) [reply docs/src/rsh/errors/index.md:2041]: I thought that `ParticipantClass` was not to be removed?

[the remark this answers is not in the document]
Comment (jeapostrophe) [reply docs/src/rsh/errors/index.md:2041]: Correct, we need to continue mentioning all of them. Even though we don't want to encourage people to use them, we have to explain the error completely.

**Review by hamirmahal — CHANGES_REQUESTED** [review]

File: hs/t/y/Fix_Part_DuplicatePart.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+Verifying knowledge assertions
+Verifying for generic connector
+  Verifying when ALL participants are honest
+  Verifying when NO participants are honest
+Checked 7 theorems; No failures!
```
[the remark this answers is not in the document]
Comment (EBEREGIT) [reply hs/t/y/Fix_Part_DuplicatePart.txt:5]: I still got the same output even though I used the new script. Let's see if it passes this time around.

File: docs/src/rsh/errors/index.md [hunk 3]
```diff
@@ -2106,11 +2106,32 @@ can either abstract the arguments into a new `{!rsh} Data` type or make separate
 
 ## {#RE0109} RE0109
```
Comment (TheChronicMonster) [inline docs/src/rsh/errors/index.md:2108]: @EBEREGIT We need to add a note under the heading:

`This error will not happen after version v0.1.`

See an example at [RE0047](https://docs.reach.sh/rsh/errors/#RE0047)

**Review by TheChronicMonster — CHANGES_REQUESTED** [review]

**Review by TheChronicMonster — APPROVED** [review]

**Pull request merged**